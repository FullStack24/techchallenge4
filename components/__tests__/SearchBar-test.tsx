import * as React from 'react';
import renderer from 'react-test-renderer';
import SearchBar from '../SearchBar';

it('deve renderizar o SearchBar corretamente', () => {
  const mockOnSearch = jest.fn();
  const tree = renderer.create(
    <SearchBar searchTerm="test" onSearch={mockOnSearch} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
