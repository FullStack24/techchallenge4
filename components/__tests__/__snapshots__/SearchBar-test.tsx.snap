// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deve renderizar o SearchBar corretamente 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "borderRadius": 12,
      "flexDirection": "row",
      "marginBottom": 16,
      "paddingHorizontal": 6,
    }
  }
>
  <Text />
  <TextInput
    onChangeText={[MockFunction]}
    placeholder="Buscar..."
    placeholderTextColor="#A9A9A9"
    style={
      {
        "color": "#000000",
        "flex": 1,
        "fontSize": 12,
        "height": 30,
      }
    }
    value="test"
  />
</View>
`;
